src/mmu/vram.rs

```diff
@@ -0,0 +1,35 @@
+use std::ops::{Index, IndexMut};
+
+const DMG_VRAM_SIZE: usize = 1 << 13; //  8 192
+const CBG_VRAM_SIZE: usize = 1 << 14; // 16 384
+const BANK_WIDTH: usize = 1 << 13;
+
+pub struct VRAM {
+    data: Vec<u8>,
```

Comment:
Since the size should not be changed, a Box<[u8]> might be more appropriate.
If you make the change, add .into_boxed_slice() after the vec![] in the new() method

Reply:
8e157c8